tests/test_backends.py

```diff
@@ -587,3 +592,45 @@ def testAcrossVariantSets(self):
                         self.verifySearchVariants(
                             variantSets, referenceName, 0, 2 ** 32,
                             pageSize=pageSize)
+
+
+class TestCallSets(TestWormtableBackend):
+    """
+    Tests the searchCallSets end point
+    """
+    def testCallSetIds(self):
+        # verify that callSetIds are equal to sample columns in wormtable
+        for variantSetId in self.getVariantSetIds():
+            callSetIds = self.getCallSetIds(variantSetId)
+            variantSets = self._backend._variantSetIdMap[variantSetId]
```

Comment:
We're also breaking isolation here. If we need to get these values we should add a method to do it.